kie-server-parent/kie-server-services/kie-server-services-jbpm-ui/src/main/resources/form-templates-providers/js/kieserver-ui.js

```diff
@@ -556,3 +556,11 @@ function closeCreationForm(fieldId) {
 	currentRow = null;
 }
 
+function getLocalDateWithoutTime(elementId) {
+	console.log('LocalDateWithoutTime Element =' + document.getElementById(elementId).value);
```

Comment:
@bxf12315  could this console.log be removed?

Reply:
Done